### PR TITLE
feat: Get event rundown

### PR DESCRIPTION
**Pull request by nataniadeandra**
**Changes added:**
Implemented test cases and logic to get event rundown

### CODE REVIEW

**Review by catherineeangel — APPROVED** [review]
LGTM

File: rundown/views.py [hunk 1]
```diff
@@ -83,4 +85,12 @@ def patch(self, request, id):
         updated_rundown.description = new_description
         updated_rundown.save()
         rundown_serializers = RundownSerializer(updated_rundown)
-        return Response(rundown_serializers.data, status=200) 
\ No newline at end of file
+        return Response(rundown_serializers.data, status=200) 
+
+class RundownListView(generics.ListAPIView):
+    permission_classes = [IsAuthenticated]
```
Comment (emiriko) [inline rundown/views.py:91]: Is isAuthenticated enough for this? I think only the owner should access (event owner that can access this endpoint). What do you think?

**Review by emiriko — CHANGES_REQUESTED** [review]
Overall nice implementation, However, I think we could make the endpoint more secure. Good job!

File: rundown/views.py (hunk 1 above)
Comment (JohannesSetiawan) [reply rundown/views.py:91]: Yeah, I agree with Alvaro. It is better if only the creator of the event can access this rundown.

**Review by JohannesSetiawan — APPROVED** [review]
Overall looks good, but some things can be changed to make a more secure backend for seeing event timeline.

File: rundown/views.py (hunk 1 above)
Comment (nataniadeandra) [reply rundown/views.py:91]: Will modify!

Comment (nataniadeandra) [reply rundown/views.py:93]: Have modified! Please check @emiriko @JohannesSetiawan 😃

**Review by emiriko — APPROVED** [review]
LGTM

**Review by JohannesSetiawan — APPROVED** [review]
Changes approved. LGTM!

**Pull request merged**